src/SourceBuild/content/test/Microsoft.DotNet.UnifiedBuild.Tasks.Tests/Microsoft.DotNet.UnifiedBuild.Tasks.Tests.csproj

```diff
@@ -0,0 +1,62 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetCurrent)</TargetFramework>
+    <DefaultExcludesInProjectFolder>$(DefaultExcludesInProjectFolder);assets/**/*</DefaultExcludesInProjectFolder>
+    <VSTestLogger>console%3bverbosity=normal;trx%3bverbosity=diagnostic%3bLogFileName=$(MSBuildProjectName).trx</VSTestLogger>
+    <VSTestCLIRunSettings>$(VSTestCLIRunSettings);RunConfiguration.DotNetHostPath=$(DotnetTool)</VSTestCLIRunSettings>
+    <!--
+      Required while we're using direct SDK assembly references in
+      $(RepositoryEngineeringDir)tools/Directory.Build.props for all tools projects
+      including Microsoft.DotNet.UnifiedBuild.Tasks.
+    -->
+    <NoWarn>$(NoWarn);MSB3277</NoWarn>
```

Comment:
We removed this part already from main from the test project that you probably copied.